src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -28,7 +28,8 @@ class AlternativesDialog < UI::Dialog
     MIN_WIDTH = 60
     MIN_HEIGHT = 20
 
-    def initialize
+    def initialize(alternative)
+      @alternative = alternative
       @mock_slaves = {
```

Comment:
Yes, that was used to mock the slaves which are not implemented yet. But now doesn't work.